test/SILOptimizer/package-cmo.swift

```diff
@@ -193,6 +193,22 @@ package func getPkgModuleKlassMemberTBD() -> Int {
   return ModuleTBD.pkgModuleKlassMember()
 }
 
+// CHECK-LABEL: sil package @$s4Main23callsPackageClassMethod1cSi6Module0cD0C_tF : $@convention(thin) (@guaranteed PackageClass) -> Int
+// CHECK-NOT:     class_method
```

Comment:
This file contains tests in non-resilient mode; could you also add cases to package-cmo-resilient-mode.swift?